pkgs/by-name/sl/slint-lsp/package.nix

```diff
@@ -46,10 +46,13 @@ rustPlatform.buildRustPackage rec {
     OpenGL
   ];
 
+  doCheck = false;
+
   postInstall = lib.optionalString stdenv.hostPlatform.isLinux  ''
     patchelf --set-rpath ${lib.makeLibraryPath rpathLibs} $out/bin/slint-lsp
   '';
 
+  # Tests requires `i_slint_backend_testing` which is only a dev dependency
   dontPatchELF = true;
```

Comment:
You inserted the comment for the wrong attribute :sweat_smile:

Reply:
woops, rough morning today!